app/views/groups/new_subgroup.html.erb

```diff
@@ -13,11 +13,11 @@
           authorized_namespaces: @authorized_namespaces
         } %>
 
-            <div class="my-2">
-              <%= form.submit t(:'groups.new_subgroup.submit'), class: "btn btn-primary mr-1" %>
-              <%= link_to t(:'groups.new_subgroup.cancel'), group_path(@group), class: "btn btn-default" %>
-            </div>
-        <% end %>
+        <div class="my-2">
+          <%= form.submit t(:'groups.new_subgroup.submit'), class: "button button--state-primary button--size-default mr-1" %>
+          <%= link_to t(:'groups.new_subgroup.cancel'), group_path(@group), class: "button button--state-default button--size-default w-full" %>
```

Comment:
Same as previous comment. One of the buttons is full width.
![image](https://github.com/phac-nml/irida-next/assets/325703/b531dc38-5192-4f56-aef6-5f4977530013)